crypto/armcap.c

```diff
@@ -23,6 +23,7 @@
 unsigned int OPENSSL_armcap_P = 0;
 unsigned int OPENSSL_arm_midr = 0;
 unsigned int OPENSSL_armv8_rsa_neonized = 0;
+unsigned int OPENSSL_armv8_aes_unroll8_eor3 = 0;
```

Comment:
Please don't add more of theses feature bits outside the OPENSSL_armcap_P,
even if it's only a feature that would trap if not available, but more like a hint that about
what would be faster, I still would recommend that there are bits in the `OPENSSL_armcap`
environment variable.
This is request is mostly about testability, since this switches code paths on or off dependent
on the CPU micro-architecture, but why shouldn't we have the ability to test other code paths
that may be non-optimal for the used CPU?
(unfortunately the OPENSSL_arm_midr and OPENSSL_armv8_rsa_neonized made their
way into 3.0, but I would rather like them to be converted to OPENSSL_armcap_P bits as well)

Reply:
assign the bit 11 for ARMV8_UNROLL8_EOR3. Done.